server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -958,7 +958,8 @@ private TabletMetadata commitCompaction(TCompactionStats stats, ExternalCompacti
         // ELASTICITY_TODO check return value and retry, could fail because of race conditions
         var result = tabletsMutator.process().get(extent);
         if (result.getStatus() == Ample.ConditionalResult.Status.ACCEPTED) {
-          // compaction was committed
+          // compaction was committed, mark the compaction input files for deletion
+          ctx.getAmple().putGcCandidates(extent.tableId(), ecm.getJobFiles());
```

Comment:
Could add this comment.


```suggestion
          // ELASTICITIY_TODO in the case of process death the GC candidates would never be added like #3811.  If
          // compaction commit were moved to FATE per #3559 then this would not be an issue.  If compaction
          // commit is never moved to FATE, then this addition could moved to the compaction refresh
          // process. The compaction refresh procss will go away if compaction commit is moved to FATE, so 
          // should only do this if not moving to FATE.
          ctx.getAmple().putGcCandidates(extent.tableId(), ecm.getJobFiles());
```